src/lib/service/hangar.ts

```diff
@@ -0,0 +1,17 @@
+export interface Result {
+  pagination: Pagination;
+}
+
+export interface Pagination {
+  limit: number;
+  offset: number;
+  count: number;
+}
+
+const fetcher = (url: string) => fetch(url).then((res) => res.json());
+const getJSON = <T>(path: string): Promise<T> => fetcher(path);
+
+export const useHangarProjects = (platform: string): Promise<Result> =>
+  getJSON(
+    `https://hangar.papermc.io/api/v1/projects?orderWithRelevance=true&limit=1&offset=0&platform=${platform.toUpperCase()}`,
+  );
```

Comment:
Since this isn't a React hook, using a `use` naming convention here doesn't make sense. A better name would be eg. `getHangerProjects`. Also, if `fetcher` and `getJSON` aren't used elsewhere, then `fetch` should be used directly.

Reply:
Adjusted this as requested, but running lint fix changed the code layout.